src/plugins/discover/public/application/main/data_fetching/fetch_esql.ts

```diff
@@ -33,9 +33,10 @@ export function fetchEsql(
   inspectorAdapters: Adapters,
   abortSignal?: AbortSignal,
   filters?: Filter[],
-  inputQuery?: Query
+  inputQuery?: Query,
+  inputTimeRange?: TimeRange
```

Comment:
I added this so that the ES|QL panels will respond to the time slider on dashboards